apps/src/sites/studio/pages/pd/pre_workshop_survey/new.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PreWorkshopSurvey from '@cdo/apps/code-studio/pd/pre_workshop_survey/PreWorkshopSurvey';
+import getScriptData from '@cdo/apps/util/getScriptData';
+
+function render() {
+  ReactDOM.render(
+    <PreWorkshopSurvey
+      {...getScriptData('props')}
+    />,
+    document.getElementById('application-container')
+  );
+}
+
+document.addEventListener("DOMContentLoaded", function (event) {
+  render();
```

Comment:
Any reason this isn't an anonymous inner function?

Reply:
No. I'm pretty sure I copied that from one of the others but now that I look they're doing it the way you suggest (which I agree is cleaner). Will update